kits/website-optimiser/apps/app/globals.css

```
@@ -0,0 +1,558 @@
+/* stylelint-disable at-rule-no-unknown scss/at-rule-no-unknown */
```

Stylelint (17.14.0)

[error] Unknown rule scss/at-rule-no-unknown. Did you mean at-rule-no-unknown?

(scss/at-rule-no-unknown)